tool/speed.cc

```diff
@@ -2341,6 +2392,24 @@ bool Speed(const std::vector<std::string> &args) {
     }
   }
 
+  bool dryrun = false;
+  if (args_map.count("-dryrun") != 0) {
+    dryrun = true;
+  }
+
+  if (dryrun) {
+    fprintf(stdout, "dryrun: %s\n", dryrun ? "true" : "false");
```

Comment:
print_json?

Reply:
Removed `-dryrun`. Current format use an array as root... Extending in a backwards compatible way is not really possible...